second.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import subprocess
+import commands
+import threading
+
+user = str(sys.argv[1])
+host_1 = str(sys.argv[2])
+host_2 = str(sys.argv[3])
+user_command = str(sys.argv[4])
+
+if (user == "" or host_1 == "" or host_2 == "" or user_command == ""):
+    print "Wrong input data"
+    exit(-1)
+hosts = [host_1, host_2]
+for host in hosts:
+    cmd = "ssh "+user+"@"+host+" '"+user_command+"'"
+    cmdOutput = commands.getoutput(cmd)
```

Comment:
1. No exit code check. 
2. What happens to the STDERR output? Can you recognize on which host the command failed?
3. The process running by SSH on a remote host could be long-running and generate a lot of output, say gigabytes.   your main process could be killed by OOM because you read every STDOUT into memory of the main process